nixos/modules/services/networking/syncthing.nix

```diff
@@ -653,7 +671,7 @@ in {
           ExecStart = ''
             ${cfg.package}/bin/syncthing \
               -no-browser \
-              -gui-address=${cfg.guiAddress} \
+              -gui-address=${if isUnixGui then "unix://" else ""}${cfg.guiAddress} \
```

Comment:
I read your previous comment about needing the `unix://` prefix, and it obviously works but the Syncthing docs say that a normal path is expected: https://docs.syncthing.net/users/syncthing.html#cmdoption-gui-address

Reply:
well, what do you want then?
the docs are indeed wrong, see https://forum.syncthing.net/t/patch-correctly-describe-how-to-use-unix-sockets-for-gui/20866

Reply:
No need to be defensive. I wasn't aware of that issue.